src/main/resources/META-INF/create.sql

```diff
@@ -1,24 +1,27 @@
-create sequence if not exists hibernate_sequence start with 1 increment by 1;
-
-create table if not exists DeviceMember (id bigint not null, name varchar(255), parent_id bigint, primary key (id) );
- 
-create table if not exists SecurityGroup ( id bigint not null, name varchar(255), policyId varchar(255), tag varchar(255), parent_id bigint, primary key (id) );
-
-create table if not exists SecurityGroupInterface ( id bigint not null, name varchar(255), policyId varchar(255), tag varchar(255));
- 
-create table if not exists VSSDevice ( id bigint not null, name varchar(255), primary key (id) );
-
-create table if not exists DOMAIN ( id bigint not null, name varchar(255), primary key (id));
-
-create table if not exists POLICY ( id bigint not null, name varchar(255), domain_fk bigint,primary key (id) );
- 
-alter table DeviceMember add constraint if not exists FK_DEVICE_MEMBER_VSS_DEVICE foreign key (parent_id) references VSSDevice;
- 
-alter table SecurityGroup add constraint if not exists FK_SECURITY_GROUP_VSS_DEVICE foreign key (parent_id) references VSSDevice;
-
-alter table POLICY add constraint if not exists FK_PO_DOMAIN foreign key (domain_fk) references DOMAIN;
-
-alter table DOMAIN add constraint if not exists DOMAIN_NAME UNIQUE (name);
-
-alter table POLICY add constraint if not exists POLICY_NAME UNIQUE (name);
-
+create sequence if not exists hibernate_sequence start with 1 increment by 1;
+
+create table if not exists SecurityGroup ( id bigint not null, name varchar(255), policyId varchar(255), tag varchar(255), parent_id bigint, primary key (id) );
+
+create table if not exists SecurityGroupInterface ( id bigint not null, name varchar(255), policyId varchar(255), tag varchar(255));
+ 
+create table if not exists DOMAIN ( id bigint not null, name varchar(255), primary key (id));
+
+create table if not exists POLICY ( id bigint not null, name varchar(255), domain_fk bigint,primary key (id) );
+
+create table if not exists DEVICE ( id bigint not null, name varchar(255), vsId bigint not null,primary key (id) );
+
+create table if not exists DEVICE_MEMBER (id bigint not null, name varchar(255), device_fk bigint, primary key (id) );
+ 
+alter table SecurityGroup add constraint if not exists FK_SECURITY_GROUP_VSS_DEVICE foreign key (parent_id) references VSSDevice;
+
+alter table POLICY add constraint if not exists FK_PO_DOMAIN foreign key (domain_fk) references DOMAIN;
+
+alter table DOMAIN add constraint if not exists DOMAIN_NAME UNIQUE (name);
+
+alter table POLICY add constraint if not exists POLICY_NAME UNIQUE (name);
+
+alter table DEVICE_MEMBER add constraint if not exists FK_DEVICE foreign key (device_fk) references DEVICE;
+
+alter table DEVICE add constraint DEVICE_NAME UNIQUE (name);
+
+alter table DEVICE_MEMBER add constraint DEVICEMEMBER_NAME UNIQUE (name);
```

Comment:
DEVICE_MEMBER_NAME also missing "if not exists" instruction in the constraint statement.